grpc/server.go

```diff
@@ -76,7 +76,7 @@ func RunServer(ctx context.Context, log logr.Logger, grpcServer *grpc.Server, po
 
 	bs := rpc.BmcService{
 		TaskRunner: taskRunner,
-		Timeout:    defaultServer.bmcTimeout,
+		Timeout:    time.Second * 60,
```

Comment:
With hard coding this timeout, it removes the ability for a user to modify the timeout at runtime.
https://github.com/tinkerbell/pbnj/blob/9a09ef8e6fd38d1e54359743a4c6a64dc598748f/cmd/server.go#L85 For our deployments internally at EM, we set this to one-minute and all rpc services use that one-minute timeout.

Reply:
Ah alright, reverted that change.
The default `15s` seemed way too low for BMC calls
https://github.com/tinkerbell/pbnj/blob/423b80894856fdcf00803e46c308f969d2db03be/grpc/server.go#L59 

In this case theDell BMCs were taking upto 50s or so to add/update users.